docs/http_grpc_api.rst

```diff
@@ -43,12 +43,18 @@ for each protocol.
 * :ref:`section-api-inference`: The inference API that accepts model
   inputs, runs inference and returns the requested outputs.
 
+GRPC only endpoint:
```

Comment:
"The inference server also exposes an endpoint based on GRPC streams <or whatever the correct term is> that is only available when using the GRPC protocol:"